### PR TITLE
chore(repo): migrate nx to latest version (13.9.5)

### PR DESCRIPTION
**Pull request by SebasG22**
## What it does

Update Nx to the latest version 

## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by SebasG22** [comment]
Hey @leosvelperez, can you take a look?

File: package.json [hunk 1]
```diff
@@ -22,15 +21,16 @@
     "@commitlint/config-conventional": "~16.2.1",
     "@commitlint/cz-commitlint": "~16.2.1",
     "@nrwl/cli": "13.8.2",
```
Comment (leosvelperez) [inline package.json:23]: Please remove `@nrwl/cli`, it's no longer needed there.

**Review by leosvelperez — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Just left a couple of small things to address and should be good to go 👍🏻

File: package.json [hunk 2]
```diff
@@ -47,6 +47,7 @@
     "inquirer": "~8.2.0",
     "jest": "^27.2.3",
     "lint-staged": "~12.3.4",
+    "nx": "13.9.3",
```
[the remark this answers is not in the document]
Comment (SebasG22) [reply package.json]: Done. Thanks for the suggestion

**Comment by leosvelperez** [comment]
Yeah, I'll be updating the project to support Nx 14. I haven't merged this PR because there are issues with the deprecated `@astrojs/parser` in the current version that makes the plugin for the project graph not work correctly. I swapped it in a branch for the `@astrojs/compiler` but then there are some issues with it because it's distributed as ESM and the Nx CLI uses the https://www.npmjs.com/package/v8-compile-cache package which doesn't play well with ESM.

I'm trying to get that sorted out and when it's done, I'll close this PR in favor of another one adding support for Nx v14.

**Comment by leosvelperez** [comment]
Actually, I went ahead and created the PR to update to the latest version of Nx (v14.1.1). I figured the issue with the project graph plugin is not depending on the Nx version upgrade. Better to have that update that solves a separate issue https://github.com/nxtensions/nxtensions/issues/46.

Closing this in favor of #47.

@SebasG22 thanks for the contribution and apologies for the delay!

**Pull request closed without merging**